src/canvas-extractor/canvas_extractor/__main__.py

```diff
@@ -181,11 +183,26 @@ def _get_submissions(
 def _get_enrollments(sync_db: sqlalchemy.engine.base.Engine) -> None:
     logger.info("Extracting Enrollments from Canvas API")
     (sections, _) = results_store["sections"]
+    (enrollments, udm_enrollments) = extract_enrollments(sections, sync_db)
     write_multi_csv(
-        extract_enrollments(sections, sync_db),
+        udm_enrollments,
         datetime.now(),
         os.path.join(output_directory, SECTION_ASSOCIATIONS_ROOT_DIRECTORY),
     )
+    results_store["enrollments"] = (enrollments, udm_enrollments)
+
+
+@catch_exceptions
+def _get_grades() -> None:
+    logger.info("Extracting Grades from Canvas API")
+    (enrollments, udm_enrollments) = results_store["enrollments"]
+    (sections, _) = results_store["sections"]
+    (_, udm_grades) = extract_grades(enrollments, udm_enrollments, sections)
```

Comment:
This is the only call to `extract_grades` that I see in this PR. Can the `all_grades` output be removed so that only one object (`udm_grades`) is returned?

Reply:
✔️